### ISSUE TITLE
Bug in "_sections.less" mixins: missing rules and incorrect default variables

### ISSUE THREAD

**Issue by dmytro-ch** [magento/magento2#18729]
There are couple of issues when trying to use mixins for "tabs" and "accordion" components.

The issue is related to `lib/web/css/source/lib/_sections.less`.
The default variables are predefined in `lib/web/css/source/lib/variables/_sections.less`.

### Summary (*)
When you try to customize the view of "tabs" and "accordion" components by redefining the default variables in the scope of a custom theme, the following issues appear:

1. The text color for active "accordion" control is not applied.
2. The text-decoration for active control is not working for both "accordion" and "tabs" components.
3. There are also incorrectly defined default variables for mixins. (e.g. accordion control visited and tab control hover variables etc.).

### Examples (*)
Let's try to define the following variables in the scope of a custom theme.

```
@tab-control__active__background-color: yellow;
@tab-control__active__color: red;
@tab-control__active__text-decoration: underline;
```

Assuming that those variables are not defined elsewhere (in the scope of current or parent theme) except the default lib variables file `lib/web/css/source/lib/variables/_sections.less`.

### Expected Result
There are red color, yellow background and underline text decoration applied for both accordion and tabs active controls.

<img width="363" alt="screen shot 2018-10-21 at 8 41 00 pm" src="https://user-images.githubusercontent.com/11693779/47270654-c0f66d80-d577-11e8-8724-ea853a85df76.png">
<img width="558" alt="screen shot 2018-10-21 at 8 52 33 pm" src="https://user-images.githubusercontent.com/11693779/47270655-c0f66d80-d577-11e8-98ac-01be9ca5a1df.png">

### Actual Result
The red color is not applied accordion active controls.
The underline text decoration is not apllied for both accordion and tabs active controls.

<img width="317" alt="screen shot 2018-10-21 at 9 01 22 pm" src="https://user-images.githubusercontent.com/11693779/47270653-bb008c80-d577-11e8-905a-2837da216b97.png">
<img width="555" alt="screen shot 2018-10-21 at 9 00 35 pm" src="https://user-images.githubusercontent.com/11693779/47270652-bb008c80-d577-11e8-9c69-716116572e77.png">

### Proposed solution
Add missing rules to `lib/web/css/source/lib/_sections.less` and change the default variable names to the correct ones.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [ ] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@dmytro-ch do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by sidolov** [comment]
Hi @dmytro-ch. Thank you for your report. 
The issue has been fixed in magento/magento2#18730 by @dmytro-ch in 2.3-develop branch
Related commit(s):
 - [19a2d2f2a2156b96733fadc88e16d07e871aadf7](https://github.com/magento/magento2/commit/19a2d2f2a2156b96733fadc88e16d07e871aadf7)
 - [310afd896e0c3cc82d81e50f92dff928c231f2bc](https://github.com/magento/magento2/commit/310afd896e0c3cc82d81e50f92dff928c231f2bc)

The fix will be available with the upcoming 2.3.1 release.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your report. 
The issue has been fixed in magento/magento2#18875 by @gelanivishal in 2.2-develop branch
Related commit(s):
 - [999e79f85c81fb49d5c90732bc275afda36f854a](https://github.com/magento/magento2/commit/999e79f85c81fb49d5c90732bc275afda36f854a)
 - [5333810773c2d8160b6c234ac27f8e09edb1382c](https://github.com/magento/magento2/commit/5333810773c2d8160b6c234ac27f8e09edb1382c)

The fix will be available with the upcoming 2.2.8 release.
